### PR TITLE
feat(APP-3298): Add support for ZkSync mainnet

### PR DESCRIPTION
**Pull request by sepehr2github**
## Description

- ZkSync support added

Task: [APP-3289](https://aragonassociation.atlassian.net/browse/APP-3289)

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I have tested my code on the test network.

[APP-3289]: https://aragonassociation.atlassian.net/browse/APP-3289?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/hooks/useWallet.tsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ export const useWallet = (): IUseWallet => {
 
   const provider = useMemo(
     () =>
-      ['mumbai', 'polygon'].includes(network)
+      ['mumbai', 'polygon', 'zkSepolia'].includes(network)
```
Comment (cgero-eth) [inline src/hooks/useWallet.tsx]: Why is this needed?

**Review by cgero-eth — COMMENTED** [review]
Thanks for the update @sepehr2github! I only left a question on the PR

File: src/hooks/useWallet.tsx (hunk 1 above)
Comment (sepehr2github) [reply src/hooks/useWallet.tsx]: Nice catch. I haven't changed this intentionally. probably changed after the merge

**Review by cgero-eth — APPROVED** [review]
LGTM, thanks @sepehr2github!
